crates/bevy_pbr/src/environment_map/filter_coefficents.rs

```diff
@@ -0,0 +1,3264 @@
+// Original source: https://www.activision.com/cdn/research/coeffs_quad_32.txt
```

Comment:
This file is $200$ KB of text to store $4 * 4 * 24 * 3 * 5 * 7 = 40$ KB of data...

Reply:
Hmmm. I supposed I could serialize it to binary and then use include_bytes!() on it, to reduce the size in the repo.

Reply:
Deleting all the spaces brings it down to 141KB from 199KB 😅

Reply:
Blame rustfmt :shrug:

Reply:
Left the rust file in the repo, but removed the module so it doesn't slow down rust-analyzer/compile times. It now uses include_bytes() to load a binary file instead.